.github/workflows/sync-tests.yml

```diff
@@ -0,0 +1,47 @@
+name: sync-tests
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+      - '*rc'
+      - 'regenesis/*'
```

Comment:
It looks like we have branch creep

Reply:
@snario we've agreed to move away from `*rc` right? The other 3 are correct otherwise @tynes